spec/fixtures/files/datahashes/core/media1.json

```diff
@@ -8,14 +8,15 @@
     "measuredPart": "framed",
     "dimensionSummary": "Past is gone",
-    "dimension": "base;weight;;circumference",
```

Comment:
Important thing #1 about this file: changed the **field group** value delimiter (`;`) to the **field subgroup** value delimiter (`^^`). See [this CollectionSpace record](https://anthro.dev.collectionspace.org/cspace/anthro/record/collectionobject/db433216-166e-4fe7-b795) and [its corresponding datahash](https://github.com/collectionspace/collectionspace-mapper/blob/master/spec/fixtures/files/datahashes/anthro/collectionobject2.json) for an example of repeated `measuredPartGroup` instances, each containing repeated `dimensionSubgroup` instances.

Reply:
Okay.